atomic_defi_design/qml/Exchange/Trade/SimpleView/Main.qml

```diff
@@ -16,6 +16,7 @@ Item {
     readonly property var subPages: Main.getSubPages()
 
     // Variable which holds the current sub-page of the SimpleView.
+    property string recover_funds_result: '{}'
```

Comment:
camelCase

Reply:
Also the comment is misplaced, it should be on the upper one property, don't know why it's there now

Reply:
old code needed by refund, not writted by me, it will be change later.